manager/ui/war/tomcat8/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,6 @@
+<Context path="/apimanui">
```

Comment:
I wish we could do this Keycloak config as an external thing somehow instead of it being stuck inside of the WAR. I wonder if it's possible?

Reply:
https://stackoverflow.com/questions/13956651/externalizing-tomcat-webapp-config-from-war-file